pylearn2/datasets/dense_design_matrix.py

```diff
@@ -264,13 +264,14 @@ def _check_labels(self):
             assert self.y.ndim <= 2
             assert np.all(self.y < self.y_labels)
 
-    @functools.wraps(Dataset.iterator)
+    # Cannot wrap the iterator method cause it is abstract
+    # @functools.wraps(Dataset.iterator)
```

Comment:
This means that the docstring does not appear in the documentation anymore. It will probably trigger a test failure in the documentation tester.